raster-foundry/raster-foundry
# Prevent multiple active roles on the same group from being created

## Overview
See title

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo
![image](https://user-images.githubusercontent.com/4392704/41872606-6244d2ca-7890-11e8-8dca-4812c5f81dda.png)

## Testing Instructions

* Run the migration
* In psql, verify that you cannot create two `user_group_role`'s for the same group and user if they are both active
* In psql, verify that setting is_active to false will allow you to create a new UGR

Closes #3492
